components/Style/ArticleDetails.styled.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import css from "styled-jsx/css";
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledArticle = styled.article`
+  position: relative;
+  margin-bottom: 50px;
+`;
+
+const StyledListItem = styled.li`
+  list-style: none;
```

Comment:
The list-style property should be applied to the list `<ul>`,` <ol>`
```suggestion
  list-style: none;
```